### PR TITLE
Clean up Grants page and refs

### LINKED ISSUE #3592: Grants info in the wiki is outdated
The link [here](https://wiki.polkadot.network/docs/build-index#grants) doesn't point to what it describes, and the information in the [grants page](https://wiki.polkadot.network/docs/grants) is outdated (we don't have limits for follow-up applications, links are outdated etc). This is duplicate info from our README so it's always gonna be hard to keep it in sync.

I would delete the page and send people to the repo/new grants website when we have it rom the first link.

### PR DESCRIPTION
**Pull request by DrW3RK**
The grants page has to be made generic, providing information on all the funding sources available in the ecosystem

### CODE REVIEW

File: docs/general/grants.md
Comment (alxs) [inline docs/general/grants.md]:
```suggestion
Web3 Foundation offers grants for open source software development and research around Substrate,
Polkadot, Kusama and ink!. Applications and deliveries are tracked transparently on GitHub. Information 
regarding requirements, the application process, deliveries, etc. can be found in the
[Grants Program README](https://github.com/w3f/Grants-Program#web3-foundation-grants-program). For guidance, there is also a list of
[previously accepted applications](https://github.com/w3f/Grants-Program/blob/master/docs/accepted_grant_applications.md#accepted-grant-applications-)
and a list of [frequently asked questions](https://github.com/w3f/Grants-Program/blob/master/docs/faq.md).
```

**Comment by alxs** [comment]
The list of alternative funding sources has been updated, see https://github.com/w3f/Grants-Program#rocket-alternative-funding-sources

**Comment by DrW3RK** [comment]
Added a link to the alternative funding sources as well.

**Comment by alxs** [comment]
The list of other grants programs is already in this page, but it's outdated.

The whole "Alternative Funding Sources" section, which is copied from our readme into this page, has been updated. I suggest just copy-pasting the contents, maybe you want to change one or two things.

**Comment by DrW3RK** [comment]
@alxs makes sense. Added a note (as info) linking to the grants section on alternative section.

File: docs/general/grants.md
Comment (alxs) [inline docs/general/grants.md]:
```suggestion
section on the Web3 Foundation grants repo for comprehensive information.
```

**Comment by DrW3RK** [comment]
Approve if this PR looks good. Will merge right after

**Review by alxs — APPROVED** [review]

**Comment by alxs** [comment]
Actually. Sorry, wait a sec

File: docs/general/grants.md
Comment (alxs) [inline docs/general/grants.md:54]:
```suggestion
- [Darwinia Grants Program](https://github.com/darwinia-network/collaboration/blob/master/grant/README.md#grant-program)
- [Moonbeam Grants Program](https://moonbeam.foundation/grants/)
```

**Comment by alxs** [comment]
The fact that they haven't updated their GitHub in a while doesn't mean it's not active. They have 1 open application at the moment ;) https://github.com/darwinia-network/collaboration/pulls But feel free to double check with them if they're no longer accepting applications

**Comment by DrW3RK** [comment]
They took down the grants page from their docs which was the recent addition to the Wiki. We can have this github link stay in the Wiki (for now)

**Pull request merged**
